.circleci/config.yml

```diff
@@ -314,6 +340,16 @@ workflows:
             - ecs-deploys
           requires:
             - deploy-production-static
+          post-steps:
+            - jira/notify:
```

Comment:
Do we need an extra Jira notify here? Why's it `staging`?

Reply:
@NoelLH no we don't, sorry I must have added that by mistake. Deleted.